src/nifgen/examples/nifgen_trigger_example.py

```diff
@@ -0,0 +1,20 @@
+import nifgen
```

Comment:
please look at how other examples are structured so that they can be invoked by the CI as well.